tests/std/tests/GH_000639_nvcc_include_all/env.lst

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+PM_COMPILER="nvcc" PM_CL="--x cu -Xcompiler -Od,-EHsc,-nologo,-W4,-WX,-openmp"

```

Comment:
When I try to run this test locally without CUDA installed, I get:

```
S:\GitHub\STL\out\build\x64>python tests\utils\stl-lit\stl-lit.py ..\..\..\tests\std\tests\GH_000639_nvcc_include_all -o testing_x64.log
stl-lit.py: S:\GitHub\STL\tests\utils\stl\test\config.py:19: note: Enabling Lit feature 'long_tests' as a result of parameter 'long_tests'
stl-lit.py: S:\GitHub\STL\tests\utils\stl\test\config.py:29: note: Enabling implicitly detected Lit features 'arch_avx2', 'locale.en_US.UTF-8', 'locale.fr_CA.ISO8859-1', 'locale.fr_FR.UTF-8', 'locale.ru_RU.UTF-8', 'locale.zh_CN.UTF-8', 'msvc', 'windows'
-- Testing: 2 tests, 2 workers --
stl-lit.py: S:\GitHub\STL\tests\utils\stl\test\tests.py:185: fatal: Could not find: 'nvcc'
stl-lit.py: S:\GitHub\STL\tests\utils\stl\test\tests.py:185: fatal: Could not find: 'nvcc'
UNRESOLVED: std :: tests/GH_000639_nvcc_include_all:0 (1 of 2)
UNRESOLVED: std :: tests/GH_000639_nvcc_include_all:1 (2 of 2)
********************
Unresolved Tests (2):
  std :: tests/GH_000639_nvcc_include_all:0
  std :: tests/GH_000639_nvcc_include_all:1


Testing Time: 0.17s
  Unresolved: 2
```

"Unresolved" sounds ominous. Can we make this "skipped" when nvcc isn't detected?

Reply:
Do you think it should only be marked as skipped for `nvcc`? It was intended to be ominous to make sure the user realizes some of the tests aren't run because the compiler can't be found.

Reply:
> Do you think it should only be marked as skipped for `nvcc`? It was intended to be ominous to make sure the user realizes some of the tests aren't run because the compiler can't be found.

I wouldn't mind a hard error if `cl` or `clang-cl` can't be found, but in practice I think people will notice if a significant majority of tests are skipped. I don't expect this to be a source of confusion.

Reply:
```
-- Testing: 2 tests, 2 workers --
stl-lit.py: S:\GitHub\STL\tests\utils\stl\test\tests.py:188: warning: Could not find: 'nvcc'
SKIPPED: std :: tests/GH_000639_nvcc_include_all:0 (1 of 2)
stl-lit.py: S:\GitHub\STL\tests\utils\stl\test\tests.py:188: warning: Could not find: 'nvcc'
SKIPPED: std :: tests/GH_000639_nvcc_include_all:1 (2 of 2)

Testing Time: 0.18s
  Skipped: 2
```

Purr-fect. :smile_cat: